### DISCUSSION TITLE
How to install OpenSSL 1.1 for update checker?

### DISCUSSION THREAD

**Discussion by worker2011** [cpeditor/cpeditor#1352] [Q&A]
When I want t oupgrade the IDE, it told me this:
![problem](https://github.com/cpeditor/cpeditor/assets/87806695/182d9611-6889-40f5-9047-3226ad2ea071)

So, please tell me what should I do?

**Comment by ouuan** [comment]
Please read your screenshot.

**Reply by worker2011** [reply]
The problem is I don't how to download the OpenSSL with the correct version. Could you tell me the download-link of an acceptable version？

**Comment by ouuan** [comment]
I see the problem that many download links for Windows are not providing OpenSSL 1.1 anymore. Is that the problem? If so, you'd better state that in your question. Otherwise, I can say nothing more than the error message says.

I don't use Windows so I'm not sure how to install OpenSSL 1.1 on Windows today. But you can just manually follow the updates for now. e.g. You can watch for releases

![watch releases](https://github.com/cpeditor/cpeditor/assets/30581822/91aee0dc-b147-4dfc-ba2a-1bfe60e53ea2)

But thanks for your reminder. CP Editor needs to switch to Qt 6 and OpenSSL 3 later.
